meteor/client/lib/ModalDialog.tsx

```diff
@@ -63,11 +63,11 @@ export class ModalDialog extends React.Component<IModalDialogAttributes> {
 
 	bindKeys = () => {
 		if (this.props.show) {
-			this.sorensen.bind('Enter', this.preventDefault, {
+			this.sorensen.bind('AnyEnter', this.preventDefault, {
```

Comment:
Since we use this magic string so many times, there could be a point in extracting it into an Enum. (Not sure if it makes sense for all our different keys, but something other than magic strings would be nice)

Reply:
That is true. I would prefer if it was something that Sorensen exposed.